### PR TITLE
autofixed coding standard issues for tests

### PR DESCRIPTION
**Pull request by xalopp**
Travis CI will also check coding standard for tests.
Violations are autofixed with phpcbf.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ before_script:
 
 script:
   - (cd $PHPCS_DIR ; ../../bin/phpunit --filter MO4)
-  - $PHPCS_DIR/bin/phpcs $MO4CS_DIR/Sniffs --standard=$PHPCS_DIR/phpcs.xml.dist --report=summary -np
+  - $PHPCS_DIR/bin/phpcs $MO4CS_DIR --standard=$PHPCS_DIR/phpcs.xml.dist --report=summary -np
```
Comment (mmoll) [inline .travis.yml]: using `--report=diff` would help potential contrubutors here

Comment (xalopp) [reply .travis.yml]: I think `--report=full` (which is default) is more valuable, since the violation is explained

Comment (mmoll) [reply .travis.yml]: also 🆗 :+1:

**Comment by mmoll** [comment]
merged, merci @xalopp

**Pull request merged**